.github/workflows/end2end-test.yml

```
@@ -142,3 +142,13 @@ jobs:
                   repo-token: '${{ secrets.GITHUB_TOKEN }}'
                   label: '[Type] Flaky Test'
                   artifact-path: flaky-tests
+
+    required-e2e-playwright-status-checks:
+        name: All end-to-end tests passing.
```

```suggestion
        name: All end-to-end tests passing
```

Removing the period at the end matches the naming convention. The same applies to other status checks.